custom_components/luxtronik/climate.py

```diff
@@ -53,14 +53,25 @@
 MIN_TEMPERATURE = 8
 MAX_TEMPERATURE = 28
 
-HVAC_ACTION_MAPPING: dict[str, str] = {
+HVAC_ACTION_MAPPING_HEAT: dict[str, str] = {
     LuxOperationMode.heating.value: HVACAction.HEATING.value,
-    LuxOperationMode.domestic_water.value: HVACAction.HEATING.value,
+    LuxOperationMode.domestic_water.value: HVACAction.IDLE.value,
```

Comment:
Sorry, but I find this very strange. 
Let's say we are in summer, and outside it is 20+ degrees. 
Are you saying that when the heatpump heats the domestic hot water, the heating device is actually (also) heating your underfloor? And/or that it should show heating in that case?
That is not how my system works, and I think it would confuse a lot of users (to see the heating being turned on in summer)...
In my view, the luxtronik device is either heating, or domestic_water, or cooling. One action at a time. 
That is also what the heatpump reports in its status line: There is no heatpump status "heating + domestic water", only "heating" or "hot water".

Reply:
The luxtronik firmware reports a lot of time not really the status what it is doing. See for example: [OperationMode correction?](https://github.com/Bouni/python-luxtronik/issues/84)